docker/collector/kafka/install.sh

```diff
@@ -16,9 +16,7 @@
 set -eux
 
 echo "*** Installing Kafka and dependencies"
-apk add --update --no-cache jq curl
-
-APACHE_MIRROR=$(curl --stderr /dev/null https://www.apache.org/dyn/closer.cgi\?as_json\=1 | jq -r '.preferred')
+APACHE_MIRROR=$(curl --stderr /dev/null https://www.apache.org/dyn/closer.cgi\?as_json\=1 |sed -n '/preferred/s/.*"\(.*\)"/\1/gp')
 
 # download kafka binaries
 curl -sSL $APACHE_MIRROR/kafka/$KAFKA_VERSION/kafka_$SCALA_VERSION-$KAFKA_VERSION.tgz | tar xz
```

Comment:
we could simplify downloads further by inlining shell scripts and log config, but I think it is not an issue really